kmspb referencing the wrong pb name?

currently, the code references `google.golang.org/genproto/googleapis/cloud/kms/v1` which is pre-migration, but if I understand the migration doc (https://github.com/googleapis/google-cloud-go/blob/main/migration.md) right, it should be: `cloud.google.com/go/kms/apiv1/kmspb`

https://github.com/google/go-cloud/blob/8cddf16323d2af04a4bdd9b58fb85d9b28a1abee/secrets/gcpkms/kms.go#L49

problem is that it seems to create a conflict with other libs:
```
panic: proto: file "google/cloud/kms/v1/ekm_service.proto" is already registered
	previously from: "cloud.google.com/go/kms/apiv1/kmspb"
	currently from:  "google.golang.org/genproto/googleapis/cloud/kms/v1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
```

--- a/secrets/gcpkms/kms.go
+++ b/secrets/gcpkms/kms.go
@@ -39,6 +39,7 @@ import (
 	"sync"
 
 	cloudkms "cloud.google.com/go/kms/apiv1"
+	"cloud.google.com/go/kms/apiv1/kmspb"
 	"github.com/google/wire"
 	"gocloud.dev/gcerrors"
 	"gocloud.dev/gcp"
@@ -46,7 +47,6 @@ import (
 	"gocloud.dev/internal/useragent"
 	"gocloud.dev/secrets"
 	"google.golang.org/api/option"
-	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
 	"google.golang.org/grpc/status"
 )
 

